6.4.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk upgrade --no-cache
+RUN apk add --no-cache bash=5.0.0-r0
+RUN apk add --no-cache curl=7.66.0-r0
```

Comment:
Remove the package release numbers here (i.e. `-r0`) as packages for these releases may be removed by package maintainers after incorporating new patches, and any pipeline for this image would begin failing to build. This will have no impact on the version of the tool being installed (e.g. `5.0.0` for `bash`).